viewing_party/party.py

```diff
@@ -1,23 +1,196 @@
 # ------------- WAVE 1 --------------------
 
+from pickle import NONE
+import copy
+
 def create_movie(title, genre, rating):
-    pass
+    if not (title and genre and rating): # if title or genre or rationg is None return None
+        return None
+    
+    new_movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating
+    }
+    
+    return new_movie
+
+def add_to_watched(user_data, movie):
+    
+    for watched in user_data:
+        user_data[watched] = [movie]     
+    
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    for watchlist in user_data:
+        user_data[watchlist] = [movie]     
+    
+    return user_data

```

Comment:
same as above! we are overwriting the entire list instead of adding a movie to it! We also don't need the for loop either because we already know what key we want to append to:

```suggestion
    user_data["watchlist"].append(movie)
    
    return user_data
```

Reply:
Yes, I see what you mean. Very helpful. Thank you, Claire!